src/main/java/vg/civcraft/mc/civmodcore/events/MaterialNamesLoadEvent.java

```diff
@@ -0,0 +1,22 @@
+package vg.civcraft.mc.civmodcore.events;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import javax.annotation.Nonnull;
+
+public final class MaterialNamesLoadEvent extends Event {
```

Comment:
This is not useful imo. Other plugins can just always assume names to be loaded when their onEnable is called.

Reply:
Huh, hadn't thought of that. But that would mean leave plugins unaware of when `ItemAPI.resetItemNames()` is called without the server starting or being reloaded.